gradle.properties

```diff
@@ -23,3 +23,8 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 
+# URL to Klaviyo server
```

Comment:
Using gradle.properties to extract some constants.

Reply:
Unrelated to comment is there going to be directions on how to override?

Reply:
the override goes in local.properties and I believe the instructions are in the flavor config, as well is our wiki